### ISSUE TITLE
NVDA doesn't announce button after user interaction if the content of the button was read automatically before.

### ISSUE THREAD

**Issue by leoagm** [nvaccess/nvda#9938]
I have a modal with a close button and a message. Once the modal get's open, the focus it's on the wrapper of the modal. NVDA is set to read automatically.

**Comment by Adriani90** [comment]
@leoagm can you still reproduce this in NVDA 2020.1 and Firefox 76? Could you please post a minimal test case or a webseite where we can reproduce this? Thanks.

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since years, I am closing this issue as abandoned. If you are still having it, please comment with a minimal test case and we can reopen.
